modules/virt-adding-static-public-key-web.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+[id="adding-static-public-key-web_{context}"]
+= Adding a static public key by using the web console
+
+Add a static public SSH key to a virtual machine (VM) by using the {product-title} web console. The key is saved as a project secret.
+
+.Prerequisites
+
+* You generated an SSH key pair by running the `ssh-keygen` command.
+* The VM has never booted.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
+. Create a VM, ensuring that you clear the *Start this VirtualMachine after creation* checkbox.
```

Comment:
What about adding the key during creation of the VM with the wizard? It could be described too.

Reply:
I will check that. It wasn't there when I tried creating a VM a while ago with catalog templates but I think I saw a new field there. If so, I will update the doc.

Reply:
I just checked the 4.13 and 4.14 "Create VM" dialogs.

There is no way to add an SSH key in either UI when you create a VM from the Catalog.

Reply:
Sorry, nevermind. I think older versions had this but 4.13+ does not.